src/fidesops/models/privacy_request.py

```diff
@@ -168,6 +183,9 @@ def trigger_policy_webhook(self, webhook: WebhookTypes) -> None:
         Pre-Execution webhooks send headers to the webhook in case the service needs to send back instructions
         to halt.  To resume, they use send a request to the reply-to URL with the reply-to-token.
         """
+        # temp fix for circular dependency
+        from fidesops.service.connectors import HTTPSConnector, get_connector
```

Comment:
not ideal, but this will take some time to address

Reply:
ideally we shouldn't be making outbound calls to services from db models

Reply:
maybe we move this method to the privacy request runner?

Reply:
I'll add a follow-up ticket for this, since I want to keep this PR from getting too large

Reply:
here it is: https://github.com/ethyca/fidesops/issues/134